conf/locale/en/LC_MESSAGES/django-partial.po

```diff
@@ -391,10 +391,14 @@ msgid ""
 "component."
 msgstr ""
 
-#: mentoring/questionnaire.py:73
+#: mentoring/mrq.py:76
 msgid "General feedback provided when submiting"
 msgstr ""
 
+#: mentoring/mcq.py:56
+msgid "General feedback provided when submitting, it is not shown if question selected by user its own feedback tip."
+msgstr ""
+
```

Comment:
As discussed, you don't need to update the .po file, so I'd rather it not even show up in the diff.